packages/engine/Source/Scene/Scene.js

```diff
@@ -162,6 +163,7 @@ function Scene(options) {
   this._frameState.scene3DOnly = defaultValue(options.scene3DOnly, false);
   this._removeCreditContainer = !hasCreditContainer;
   this._creditContainer = creditContainer;
+  this._fpsTracker = new FpsTracker();
```

Comment:
Since we are accessing this from another class, can we expose this as `fpsTracker` with a getter? It can be marked as `@private` so it doesn't show up in the developer API documentation.

Reply:
updated with this change :+1: